src/main/java/com/notably/logic/suggestion/SuggestionCommand.java

```diff
@@ -0,0 +1,15 @@
+package com.notably.logic.suggestion;
+
+import com.notably.model.Model;
+
+/**
+ * Represents the Suggestion Command instance.
+ */
+interface SuggestionCommand {
+    /**
+     * Gets a list of suggestions (paths or notes that match the current input).
+     * Adds this list to Suggestion Model.
```

Comment:
Nit: Rather than having a description that sounds like a two-step process, i.e. gets the list first, then add to the model, should we just simplify this to something like "Adds a list of generated suggestions to the app's model"?